src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfFindRefs.xml

```diff
@@ -24,8 +24,8 @@
       <WaitForCodeMarkers />
       <ExecuteCommand Command="Window.ActivateDocumentWindow" />
 
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="NoBasic" />
+      <!--<VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
```

Comment:
Perhaps we should just change 'VerifyRoslynModulesLoadedStatus' to stop failing in such cases? That way we will only need to change one place when we re-enable this check?